packages/lib/src/components/Card/components/CardInput/components/Installments/GetInstallmentsComp.tsx

```diff
@@ -0,0 +1,12 @@
+import { h } from 'preact';
+import Installments from './Installments';
+
+export const getInstallmentsComp = ({ amount, brand, installmentOptions, handleInstallments, showAmountsInInstallments }) => (
+    <Installments
+        amount={amount}
+        brand={brand}
+        installmentOptions={installmentOptions}
+        onChange={handleInstallments}
+        type={showAmountsInInstallments ? 'amount' : 'months'}
+    />
+);
```

Comment:
Out of curiosity, why not just render the `<Installments />` on both `CardFieldsWrapper` and `StoredCardFieldsWrapper` component instead of creating this helper function to render it?

Reply:
I was just porting something that was already there. But you're right we don't really gain anything from that util now we've split the storedCard and card components into separate files - so I'll remove the util.